
\documentclass[10pt,a4paper,sans]{moderncv} % Font sizes: 10, 11, or 12; paper sizes: a4paper, letterpaper, a5paper, legalpaper, executivepaper or landscape; font families: sans or roman

\moderncvstyle{classic} % CV theme - options include: 'casual' (default), 'classic', 'oldstyle' and 'banking'
\moderncvcolor{blue} % CV color - options include: 'blue' (default), 'orange', 'green', 'red', 'purple', 'grey' and 'black'

\usepackage{hyperref}
\usepackage{graphicx}
\usepackage{multicol}

\usepackage[scale=0.8]{geometry} % Reduce document margins
\setlength{\hintscolumnwidth}{1.5cm} % Uncomment to change the width of the dates column
\setlength{\makecvtitlenamewidth}{10cm} % For the 'classic' style, uncomment to adjust the width of the space allocated to your name


\newcommand{\curie}[2]{\href{https://bioregistry.io/#1:#2}{{#1:#2}}}
\newcommand{\wikidata}[2]{\href{https://bioregistry.io/wikidata:#1?provider=scholia}{{#2}}}

\begin{document}

{\Huge Charles Tapley Hoyt, Ph.D.}

\vspace{3mm}

\href{https://cthoyt.com}{cthoyt.com}
    $\cdot$\href{https://linkedin.com/in/cthoyt}{\includegraphics[scale=0.055]{img/LI-In-Bug}\
    linkedin.com/in/cthoyt}
    $\cdot$\href{https://github.com/cthoyt}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px}\
    github.com/cthoyt}
    $\cdot$\href{https://bioregistry.io/wikidata:Q47475003}{\includegraphics[scale=0.01]{img/wikidata_logo}\ Q47475003}
$\cdot$
\href{https://orcid.org/0000-0003-4423-4370}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}\ https://orcid.org/0000-0003-4423-4370}
$\cdot$
\href{https://scholar.google.com/citations?user=PjrpzUIAAAAJ}{\includegraphics[scale=0.015625]{img/google_scholar_icon_130918} PjrpzUIAAAAJ}

\section{Fields of Work}\label{sec:fields}

    \wikidata{Q128570}{Bioinformatics},     \wikidata{Q25303877}{Pathway Analysis},     \wikidata{Q2539}{Machine Learning},     \wikidata{Q30642}{Natural Language Processing},     \wikidata{Q324254}{Ontology},     \wikidata{Q33002955}{Knowledge Graph},     \wikidata{Q50327360}{Proteochemometrics},     \wikidata{Q5308921}{Drug Repositioning},     \wikidata{Q815297}{Systems Biology},     \wikidata{Q910164}{Cheminformatics}

\section{Education}\label{sec:education}
\cventry{2018--19}{Doctor of Philosophy}{Computational Life Sciences}{University of Bonn}{}{Thesis: \href{https://github.com/cthoyt/doctoral-thesis}{\textit{Generation and Applications of Knowledge Graphs in Systems and Networks Biology}}\\Research Advisor: Prof. Dr. Martin Hofmann-Apitius\\ GPA: 1,3 (Magna Cum Laude)}
\cventry{2015--17}{Master of Science}{Life Science Informatics}{University of Bonn}{}{Thesis: \href{https://github.com/cthoyt/masters-thesis}{\textit{PyBEL: a Computational Framework for Biological Expression Language}}\\Research Advisor: Prof. Dr. Martin Hofmann-Apitius\\GPA: 1,6}
\cventry{2011--15}{Bachelor of Science}{Chemistry}{Northeastern University}{}{GPA: 3.95/4.0 (Summa Cum Laude)\\Research Advisor: Prof. Dr. Michael P. Pollastri}

\section{Employment}\label{sec:employment}

\cventry{2021--}{Research Fellow}{Harvard Medical School, Laboratory of Systems Pharmacology}{Boston, MA}{USA (remote)}{Biocuration, data standardization, natural language processing, data integration, and automated assembly of knowledge and models}
\cventry{2020}{Computational Biologist}{Enveda Biosciences}{Boulder, CO}{USA (remote)}{Biomedical data integration, knowledge graph construction, and analysis}
\cventry{2018--19}{Lecturer}{University of Bonn}{Bonn}{Germany}{}
\cventry{2016--19}{Research Fellow}{Fraunhofer SCAI, Dept. of Bioinformatics}{Sankt Augustin}{Germany}{Systems biology modeling and analysis with the Biological Expression Language}
\cventry{2012--15}{Teaching Assistant}{Northeastern University}{Boston, MA}{USA}{}
\cventry{2014}{\textit{in Silico} Lead Discovery Co-op}{Novartis}{Cambridge, MA}{USA}{Mechanism of action deconvolution in high-throughput black-box phenotypic screens}
\cventry{2013--14}{Molecular Informatics Internship}{Pfizer}{Cambridge, MA}{USA}{Software development and database development for cancer genomics data}
\cventry{2013}{Post-Selection Chemistry Co-op}{GlaxoSmithKline}{Waltham, MA}{USA}{Lead optimization of antivirals identified by ultra-high-throughput screening}
\cventry{2012}{Research Assistant}{Pollastri Laboratory, Northeastern University}{Boston, MA}{USA}{Medicinal synthetic chemistry for repurposing PDE4 inhibitors against \textit{T. Brucei}}

\section{Awards}
\cventry{2023}{Nominated, Excellence in Biocuration Early Career Award}{International Society of Biocuration}{}{}{}
\cventry{2015}{Bernie Lemire Award}{Department of Chemistry, Northeastern University}{}{}{}
\cventry{2011}{Presidential Scholarship}{Northeastern University}{}{}{}

\newpage

{\huge Research}

\section{Publications}

Author lists have been omitted to reduce visual clutter. The following are noted:

\begin{itemize}
\item $\dagger$ signifies first or co-first authorship
\item $\ddagger$ signifies last or senior authorship
\end{itemize}

\vspace{0.5cm}

    \begin{enumerate}
    \setcounter{enumi}{0}
    \itemsep0.5em
        \item
        $\dagger$\wikidata{Q118774152}{\textbf{Prediction and curation of missing biomedical identifier mappings with Biomappings}}.
    \textit{Bioinformatics}, 2023.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTAD130}{doi:10.1093/bioinformatics/btad130}}
    \end{enumerate}
    \begin{enumerate}
    \setcounter{enumi}{1}
    \itemsep0.5em
        \item
        (preprint) \wikidata{Q115929717}{\textbf{Democratising Knowledge Representation with BioCypher}}.
    \textit{arXiv}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.48550/ARXIV.2212.13543}{doi:10.48550/arxiv.2212.13543}}
        \item
        (preprint) \wikidata{Q118951156}{\textbf{Experimental design for causal query estimation in partially observed biomolecular networks}}.
    \textit{arXiv}, 2022.  {\scriptsize
        \href{https://bioregistry.io/arxiv:2210.13423}{arxiv:2210.13423}}}
        \item
        $\dagger$\wikidata{Q115317009}{\textbf{Unifying the identification of biomedical entities with the Bioregistry}}.
    \textit{Scientific Data}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.1038/S41597-022-01807-3}{doi:10.1038/s41597-022-01807-3}}
        \item
        \wikidata{Q118950885}{\textbf{A Simple Standard for Ontological Mappings 2022: Updates of data model and outlook}}.
    \textit{OM 2022}
        \item
        \wikidata{Q118774036}{\textbf{A review of biomedical datasets relating to drug discovery: a knowledge graph perspective}}.
    \textit{Brief Bioinform}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/BIB/BBAC404}{doi:10.1093/bib/bbac404}}
        \item
        \wikidata{Q118774118}{\textbf{ChemicalX: A Deep Learning Library for Drug Pair Scoring}}.
    \textit{KDD}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.1145/3534678.3539023}{doi:10.1145/3534678.3539023}}
        \item
        \wikidata{Q114037713}{\textbf{Integrating multi-omics data reveals function and therapeutic potential of deubiquitinating enzymes}}.
    \textit{eLife}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.7554/ELIFE.72879}{doi:10.7554/elife.72879}}
        \item
        \wikidata{Q107296776}{\textbf{Understanding the Performance of Knowledge Graph Embeddings in Drug Discovery}}.
    \textit{Artificial Intelligence in the Life Sciences}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.1016/J.AILSCI.2022.100036}{doi:10.1016/j.ailsci.2022.100036}}
        \item
        \wikidata{Q114032935}{\textbf{A Simple Standard for Sharing Ontological Mappings (SSSOM)}}.
    \textit{Database}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/DATABASE/BAAC035}{doi:10.1093/database/baac035}}
        \item
        \wikidata{Q114032944}{\textbf{Gilda: biomedical entity text normalization with machine-learned disambiguation as a service}}.
    \textit{Bioinformatics Advances}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/BIOADV/VBAC034}{doi:10.1093/bioadv/vbac034}}
        \item
        \wikidata{Q111697653}{\textbf{ProtSTonKGs: A Sophisticated Transformer Trained on Protein Sequences, Text, and Knowledge Graphs}}.
    \textit{SWAT4HCLS 2022}.  {\scriptsize \href{http://ceur-ws.org/Vol-3127/paper-13.pdf}{ceur-ws:3127:13}}}

        \item
        (preprint) \wikidata{Q118780832}{\textbf{Mondo: Unifying diseases for the world, by the world}}.
    \textit{medRxiv}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.1101/2022.04.13.22273750}{doi:10.1101/2022.04.13.22273750}}
        \item
        \wikidata{Q114037635}{\textbf{Do-calculus enables estimation of causal effects in partially observed biomolecular pathways}}.
    \textit{Bioinformatics}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTAC251}{doi:10.1093/bioinformatics/btac251}}
        \item
        (preprint) $\dagger$\wikidata{Q111337218}{\textbf{A Unified Framework for Rank-based Evaluation Metrics for Link Prediction in Knowledge Graphs}}.
    \textit{arXiv}, 2022.  {\scriptsize
        \href{https://bioregistry.io/arxiv:2203.07544}{arxiv:2203.07544}}}
        \item
        \wikidata{Q113227575}{\textbf{PyBioPAX: biological pathway exchange in Python}}.
    \textit{JOSS}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.21105/JOSS.04136}{doi:10.21105/joss.04136}}
        \item
        (preprint) \wikidata{Q111102996}{\textbf{An Open Challenge for Inductive Link Prediction on Knowledge Graphs}}.
    \textit{arXiv}, 2022.  {\scriptsize
        \href{https://bioregistry.io/arxiv:2203.01520}{arxiv:2203.01520}}}
        \item
        \wikidata{Q112063219}{\textbf{STonKGs: A Sophisticated Transformer Trained on Biomedical Text and Knowledge Graphs}}.
    \textit{Bioinformatics}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTAC001}{doi:10.1093/bioinformatics/btac001}}
        \item
        \wikidata{Q118774144}{\textbf{Ontology Development Kit: a toolkit for building, maintaining and standardizing biomedical ontologies}}.
    \textit{Database}, 2022.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/DATABASE/BAAC087}{doi:10.1093/database/baac087}}
    \end{enumerate}
    \begin{enumerate}
    \setcounter{enumi}{19}
    \itemsep0.5em
        \item
        \wikidata{Q96631308}{\textbf{Bringing Light Into the Dark: A Large-scale Evaluation of Knowledge Graph Embedding Models Under a Unified Framework}}.
    \textit{TPAMI}, 2021.  {\scriptsize \href{https://bioregistry.io/doi:10.1109/TPAMI.2021.3124805}{doi:10.1109/tpami.2021.3124805}}
        \item
        \wikidata{Q110894741}{\textbf{The role of metadata in reproducible computational research}}.
    \textit{Patterns}, 2021.  {\scriptsize \href{https://bioregistry.io/doi:10.1016/J.PATTER.2021.100322}{doi:10.1016/j.patter.2021.100322}}
        \item
        $\ddagger$\wikidata{Q118774035}{\textbf{CLEP: a hybrid data- and knowledge-driven framework for generating patient representations}}.
    \textit{Bioinformatics}, 2021.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTAB340}{doi:10.1093/bioinformatics/btab340}}
        \item
        \wikidata{Q108126839}{\textbf{A Systems Biology Approach for Hypothesizing the Effect of Genetic Variants on Neuroimaging Features in Alzheimer’s Disease}}.
    \textit{JAD}, 2021.  {\scriptsize \href{https://bioregistry.io/doi:10.3233/JAD-201397}{doi:10.3233/jad-201397}}
        \item
        $\dagger$\wikidata{Q109583780}{\textbf{PyKEEN 1.0: A Python Library for Training and Evaluating Knowledge Graph Embeddings}}.
    \textit{JMLR}, 2021.  {\scriptsize
        \href{https://bioregistry.io/arxiv:2007.14175}{arxiv:2007.14175}}}
        \item
        \wikidata{Q114032921}{\textbf{Leveraging Structured Biological Knowledge for Counterfactual Inference: A Case Study of Viral Pathogenesis}}.
    \textit{IEEE TBDATA}, 2021.  {\scriptsize \href{https://bioregistry.io/doi:10.1109/TBDATA.2021.3050680}{doi:10.1109/tbdata.2021.3050680}}
    \end{enumerate}
    \begin{enumerate}
    \setcounter{enumi}{25}
    \itemsep0.5em
        \item
        (preprint) $\dagger$\wikidata{Q96909013}{\textbf{Extension of Roles in the ChEBI Ontology}}.
    \textit{ChemRxiv}, 2020.  {\scriptsize \href{https://bioregistry.io/doi:10.26434/CHEMRXIV.12591221}{doi:10.26434/chemrxiv.12591221}}
        \item
        \wikidata{Q97063807}{\textbf{The Minimum Information about a Molecular Interaction Causal Statement (MI2CAST)}}.
    \textit{Bioinformatics}, 2020.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTAA622}{doi:10.1093/bioinformatics/btaa622}}
        \item
        \wikidata{Q98200725}{\textbf{GuiltyTargets: Prioritization of Novel Therapeutic Targets with Deep Network Representation Learning}}.
    \textit{IEEE/ACM Transactions on Computational Biology and Bioinformatics}, 2020.  {\scriptsize \href{https://bioregistry.io/doi:10.1109/TCBB.2020.3003830}{doi:10.1109/tcbb.2020.3003830}}
        \item
        \wikidata{Q96169295}{\textbf{PS4DR: a multimodal workflow for identification and prioritization of drugs based on pathway signatures}}.
    \textit{BMC Bioinf.}, 2020.  {\scriptsize \href{https://bioregistry.io/doi:10.1186/S12859-020-03568-5}{doi:10.1186/s12859-020-03568-5}}
        \item
        \wikidata{Q118952373}{\textbf{Identifying the parametric occurrence of multiple steady states for some biological networks}}.
    \textit{Journal of Symbolic Computation}, 2020.  {\scriptsize \href{https://bioregistry.io/doi:10.1016/J.JSC.2019.07.008}{doi:10.1016/j.jsc.2019.07.008}}
        \item
        \wikidata{Q90643177}{\textbf{A Computational Approach for Mapping Heme Biology in the Context of Hemolytic Disorders}}.
    \textit{Frontiers in Bioengineering and Biotechnology}, 2020.  {\scriptsize \href{https://bioregistry.io/doi:10.3389/FBIOE.2020.00074}{doi:10.3389/fbioe.2020.00074}}
    \end{enumerate}
    \begin{enumerate}
    \setcounter{enumi}{31}
    \itemsep0.5em
        \item
        \wikidata{Q91869190}{\textbf{The Impact of Pathway Database Choice on Statistical Enrichment Analysis and Predictive Modeling}}.
    \textit{Frontiers in Genetics}, 2019.  {\scriptsize \href{https://bioregistry.io/doi:10.3389/FGENE.2019.01203}{doi:10.3389/fgene.2019.01203}}
        \item
        \wikidata{Q90659732}{\textbf{Quantifying mechanisms in neurodegenerative diseases (NDDs) using candidate mechanism perturbation amplitude (CMPA) algorithm}}.
    \textit{BMC Bioinf.}, 2019.  {\scriptsize \href{https://bioregistry.io/doi:10.1186/S12859-019-3101-1}{doi:10.1186/s12859-019-3101-1}}
        \item
        \wikidata{Q72584451}{\textbf{The KEEN Universe}}.
    \textit{ISWC 2019}.  {\scriptsize \href{https://bioregistry.io/doi:10.1007/978-3-030-30796-7_1}{doi:10.1007/978-3-030-30796-7_1}}
        \item
        \wikidata{Q72621157}{\textbf{Predicting Missing Links Using PyKEEN}}.
    \textit{ISWC 2019}.  {\scriptsize \href{http://ceur-ws.org/Vol-2456/paper64.pdf}{ceur-ws:2456:64}}}

        \item
        \wikidata{Q69105793}{\textbf{RatVec: A General Approach for Low-dimensional Distributed Vector Representations via Domain-specific Rational Kernels}}.
    \textit{LWDA 2019}
        \item
        \wikidata{Q63709729}{\textbf{BioKEEN: a library for learning and evaluating biological knowledge graph embeddings}}.
    \textit{Bioinformatics}, 2019.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTZ117}{doi:10.1093/bioinformatics/btz117}}
        \item
        \wikidata{Q64054035}{\textbf{PathMe: merging and exploring mechanistic pathway knowledge}}.
    \textit{BMC Bioinf.}, 2019.  {\scriptsize \href{https://bioregistry.io/doi:10.1186/S12859-019-2863-9}{doi:10.1186/s12859-019-2863-9}}
        \item
        (preprint) $\dagger$\wikidata{Q63709723}{\textbf{Integration of Structured Biological Data Sources using Biological Expression Language}}.
    \textit{bioRxiv}, 2019.  {\scriptsize \href{https://bioregistry.io/doi:10.1101/631812}{doi:10.1101/631812}} {\scriptsize \href{https://bioregistry.io/biorxiv:631812}{biorxiv:631812}}}
        \item
        \wikidata{Q60045097}{\textbf{ComPath: an ecosystem for exploring, analyzing, and curating mappings across pathway databases}}.
    \textit{NPJ Syst Biol Appl.}, 2019.  {\scriptsize \href{https://bioregistry.io/doi:10.1038/S41540-018-0078-8}{doi:10.1038/s41540-018-0078-8}}
        \item
        $\dagger$\wikidata{Q64911025}{\textbf{Re-curation and rational enrichment of knowledge graphs in Biological Expression Language}}.
    \textit{Database}, 2019.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/DATABASE/BAZ068}{doi:10.1093/database/baz068}}
        \item
        \wikidata{Q82511885}{\textbf{Challenges of Integrative Disease Modeling in Alzheimer's Disease}}.
    \textit{Frontiers in molecular biosciences}, 2019.  {\scriptsize \href{https://bioregistry.io/doi:10.3389/FMOLB.2019.00158}{doi:10.3389/fmolb.2019.00158}}
    \end{enumerate}
    \begin{enumerate}
    \setcounter{enumi}{42}
    \itemsep0.5em
        \item
        $\dagger$\wikidata{Q60302045}{\textbf{BEL Commons: an environment for exploration and analysis of networks encoded in Biological Expression Language}}.
    \textit{Database}, 2018.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/DATABASE/BAY126}{doi:10.1093/database/bay126}}
        \item
        \wikidata{Q51056857}{\textbf{BEL2ABM: agent-based simulation of static models in Biological Expression Language}}.
    \textit{Bioinformatics}, 2018.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTY107}{doi:10.1093/bioinformatics/bty107}}
        \item
        $\dagger$\wikidata{Q42695788}{\textbf{PyBEL: a Computational Framework for Biological Expression Language}}.
    \textit{Bioinformatics}, 2018.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTX660}{doi:10.1093/bioinformatics/btx660}}
        \item
        $\dagger$\wikidata{Q55315340}{\textbf{A systematic approach for identifying shared mechanisms in epilepsy and its comorbidities}}.
    \textit{Database}, 2018.  {\scriptsize \href{https://bioregistry.io/doi:10.1093/DATABASE/BAY050}{doi:10.1093/database/bay050}}
    \end{enumerate}
    \begin{enumerate}
    \setcounter{enumi}{46}
    \itemsep0.5em
        \item
        \wikidata{Q47479157}{\textbf{A Case Study on the Parametric Occurrence of Multiple Steady States}}.
    \textit{ISSAC 2017}.  {\scriptsize \href{https://bioregistry.io/doi:10.1145/3087604.3087622}{doi:10.1145/3087604.3087622}}
    \end{enumerate}
    \begin{enumerate}
    \setcounter{enumi}{47}
    \itemsep0.5em
        \item
        \wikidata{Q34138086}{\textbf{Repurposing human PDE4 inhibitors for neglected tropical diseases: design, synthesis and evaluation of cilomilast analogues as Trypanosoma brucei PDEB1 inhibitors}}.
    \textit{Bioorg Med Chem Lett}, 2014.  {\scriptsize \href{https://bioregistry.io/doi:10.1016/J.BMCL.2014.07.063}{doi:10.1016/j.bmcl.2014.07.063}}
    \end{enumerate}

\section{Invited Presentations}
\begin{enumerate}
\itemsep0.5em
    \item \textbf{\href{https://docs.google.com/presentation/d/1oKxBzSCZGuFlIhw6jAXw00xgEGs7KgmaVR54NEdPprw/edit?usp=sharing}{Using dashboards to monitor ontology standardisation and community activity}}. \textit{Ontology Summit 2023} (February 15, 2023)

    \item \textbf{\href{https://docs.google.com/presentation/d/1Vq-tdq_PYSmKydKXEABElgt3oqYs_ridquHDLeUFL1A/edit?usp=sharing}{Introduction to WPCI 2022}}. \textit{2022 Workshop on Prefixes, CURIEs, and IRIs} (December 5, 2022)

    \item \textbf{\href{https://bit.ly/icbo2022-cth}{The Bioregistry, CURIEs, and OBO Community Health}}. \textit{International Conference on Biomedical Ontology (ICBO)} (September 26, 2022)

    \item \textbf{\href{https://bit.ly/ontologies4chem-crog}{Axiomatizing Chemical Roles}}. \textit{Ontologies4Chem Workshop 2022} (September 7, 2022)

    \item \textbf{\href{https://bit.ly/pykeen-kgc2022}{Knowledge Graph Embedding with PyKEEN in 2022}}. \textit{Knowledge Graph Conference (KGC 2022)} (May 5, 2022)

    \item \textbf{\href{https://docs.google.com/presentation/d/1D0HIX41IgVdTQYKlakWcvlZvOktX23EqbLcs3LmI7mg/edit?usp=sharing}{The Biopragmatics Stack: Biomedical and Chemical Semantics for Humans}}. \textit{Machine-Actionable Data Interoperability for Chemical Sciences (MADICES)} (February 8, 2022)

    \item \textbf{\href{https://bit.ly/WPCI2021}{Introduction to WPCI 2021}}. \textit{2021 Workshop on Prefixes, CURIEs, and IRIs} (October 29, 2021)

    \item \textbf{\href{https://www.youtube.com/watch?v=bLN1V5fZD2g&t=3665s}{Current Issues in Theory, Reproducibility, and Utility of Graph Machine Learning in the Life Sciences}}. \textit{Graph Machine Learning in Industry} (September 23, 2021)

    \item \textbf{\href{https://bit.ly/3wvP8yq}{Perspectives on Knowledge Graph Embedding Models in/out of Biomedicine}}. \textit{AstraZeneca} (April 6, 2021)

    \item \textbf{\href{https://docs.google.com/presentation/d/1C8T3XVOL8GiZ68WBXXSnI3SVjhd2aSBuVLXVPPGqKTw/edit?usp=sharing}{Future Directions for WikiPathway Meta-curation}}. \textit{WikiPathways Developers Conference Call} (January 6, 2021)

    \item \textbf{\href{https://docs.google.com/presentation/d/18ntRMvO3Q9KTX87xoLOIgqAW3iiUvxJsfEmaFh1-52A/edit?usp=sharing}{The Biological Expression Language and PyBEL in 2020}}. \textit{COVID-19 Disease Map Community Meeting} (July 10, 2020)

    \item \textbf{\href{https://youtu.be/rHhuVBpoKdI}{Introduction to the Biological Expression Language and the Rational Enrichment Workflow}}. \textit{CoronaWhy} (May 6, 2020)

    \item \textbf{\href{https://docs.google.com/presentation/d/1NQHQSD2EjOh-9XD6ffr2rDqs8AwOQ3wSBoie-e7-OXU/edit?usp=sharing}{Applications of Knowledge Graphs in Drug Discovery}}. \textit{Computational Drug Discovery Group, University of Leiden} (November 5, 2019)

    \item \textbf{\href{https://docs.google.com/presentation/d/11CG_Q3JrsqqKsDhl8uXP_TZzcyALhcVLvoP_pDmQ5JA/edit?usp=sharing}{Generation and Application of Biomedical Knowledge Graphs}}. \textit{Harvard Medical School} (July 19, 2019)

    \item \textbf{\href{https://docs.google.com/presentation/d/1Gsk8LxrVgrICproSgWZ3Gk4BXyOg-UwFlfzhVw5O0t8/edit?usp=sharing}{The PyBEL Ecosystem in 2018}}. \textit{OpenBEL Community Meeting} (May 14, 2018)

\end{enumerate}

\section{Talks and Posters}
\begin{enumerate}
\itemsep0.5em
    \item \textbf{\href{https://docs.google.com/presentation/d/1hEryoMNyp8gwt2pvPy2ErKaw_QPvnWOmzPT0Sn4Yl4s/edit?usp=sharing}{Promoting the longevity of curated scientific resources through open code, open data, and public infrastructure}}. \textit{Biocuration 2023} (April 26, 2023)

    \item \textbf{\href{https://bit.ly/glb2022-ranking-metrics}{A Unified Framework for Rank-based Evaluation Metrics for Link Prediction in Knowledge Graphs}}. \textit{Graph Learning Benchmarks (GLB 2022)} (April 26, 2022)

    \item \textbf{\href{https://f1000research.com/posters/10-1005}{Biomappings: Community Curation of Mappings between Biomedical Entities}}. \textit{4th Session of the International Society of Biocuration 2021 Virtual Conference} (October 5, 2021; poster)

    \item \textbf{\href{https://www.youtube.com/watch?v=9Pf5wr8JxCQ&t=1s}{The Bioregistry: A Metaregistry for Biomedical Entities}}. \textit{12th International Conference on Biomedical Ontologies} (September 17, 2021)

    \item \textbf{\href{https://doi.org/10.6084/m9.figshare.9917753.v3}{Maintenance and Enrichment of Disease Maps in Biological Expression Language}}. \textit{4th Disease Maps Community Meeting} (October 4, 2019; poster)

    \item \textbf{\href{https://doi.org/10.6084/m9.figshare.8288822.v2}{Identifying Drug Repositioning Candidates using Representation Learning on Heterogeneous Networks}}. \textit{The Eighth Joint Sheffield Conference on Chemoinformatics} (June 19, 2019; poster)

    \item \textbf{\href{https://docs.google.com/presentation/d/1_3j8H4f9pQ1m2ZQrF2QNfQNLh5SGqaodnl4AlaAST6w/edit?usp=sharing}{From Knowledge Assembly to Hypothesis Generation}}. \textit{Bio-IT World} (April 22, 2018)

    \item \textbf{\href{https://doi.org/10.6084/m9.figshare.6231701.v1}{Knowledge Assembly in Systems and Networks Biology}}. \textit{Bio-IT World} (April 23, 2018; poster)

    \item \textbf{\href{https://doi.org/10.6084/m9.figshare.6163775.v3}{The Human Brain Pharmacome: An Overview}}. \textit{3rd European Conference on Translational Bioinformatics} (April 17, 2018; poster)

    \item \textbf{\href{https://docs.google.com/presentation/d/1FU8lWGw6x4y9A_qB9rmxSaRRKiC0V3403EJETVGYMzk/edit?usp=sharing}{Gene Set Analysis using Phenotypic Screening Data}}. \textit{Research, Innovation and Scholarship Expo 2015} (April 9, 2015; poster)

\end{enumerate}

    \section{Research Software}\label{sec:software}
    \begin{enumerate}
    \itemsep0.5em
        \item \textbf{PyBEL}\hfill\href{https://github.com/pybel/pybel}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} pybel/pybel}
\\
    A compiler for the Biological Expression Language (BEL)

        \item \textbf{Bio2BEL}\hfill\href{https://github.com/bio2bel/bio2bel}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} bio2bel/bio2bel}
\\
    A framework for reproducible data integration in BEL and knowledge graph construction

        \item \textbf{BEL Commons}\hfill\href{https://github.com/bel-commons/bel-commons}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} bel-commons/bel-commons}
\\
    A web application for the interactive exploration of networks encoded in BEL

        \item \textbf{PyKEEN}\hfill\href{https://github.com/pykeen/pykeen}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} pykeen/pykeen}
\\
    The most expansive knowledge graph embedding framework to date

        \item \textbf{SeMRA}\hfill\href{https://github.com/biopragmatics/semra}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} biopragmatics/semra}
\\
    Automated assembly and inference of semantic mappings

        \item \textbf{Bioontologies}\hfill\href{https://github.com/biopragmatics/bioontologies}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} biopragmatics/bioontologies}
\\
    Access and processing of ontologies on top of ROBOT and OBO Graphs

        \item \textbf{PyOBO}\hfill\href{https://github.com/pyobo/pyobo}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} pyobo/pyobo}
\\
    Harmonization of biological ontologies and controlled vocabularies

        \item \textbf{curies}\hfill\href{https://github.com/cthoyt/curies}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} cthoyt/curies}
\\
    Idiomatic conversion between URIs and compact URIs (CURIEs)

        \item \textbf{GuiltyTargets}\hfill\href{https://github.com/guiltytargets/guiltytargets}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} guiltytargets/guiltytargets}
\\
    Target prioritization framework using gene expression and network representation learning

        \item \textbf{RatVec}\hfill\href{https://github.com/ratvec/ratvec}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} ratvec/ratvec}
\\
    Sequence-based representation learning

        \item \textbf{PS4DR}\hfill\href{https://github.com/ps4dr/ps4dr}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} ps4dr/ps4dr}
\\
    Drug repositioning based on bioactivity pattern matching and GWAS

        \item \textbf{SeffNet}\hfill\href{https://github.com/seffnet/seffnet}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} seffnet/seffnet}
\\
    Drug repositioning framework based on network representation learning

        \item \textbf{CLEP}\hfill\href{https://github.com/hybrid-kg/clep}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} hybrid-kg/clep}
\\
    Patient stratification framework based on network representation learning

        \item \textbf{BEL2SCM}\hfill\href{https://github.com/bel2scm/bel2scm}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} bel2scm/bel2scm}
\\
    Generation of structural causal models (SCMs) from BEL

        \item \textbf{y0 Causal Inference Engine}\hfill\href{https://github.com/y0-causal-inference/y0}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} y0-causal-inference/y0}
\\
    Representation and manipulating probabilistic expressions

        \item \textbf{STonKGs}\hfill\href{https://github.com/stonkgs/stonkgs}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} stonkgs/stonkgs}
\\
    Multimodal Transformers for biomedical text and Knowledge Graph data

        \item \textbf{ChemicalX}\hfill\href{https://github.com/AstraZeneca/chemicalx}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} AstraZeneca/chemicalx}
\\
    A deep learning library for drug-drug interaction, polypharmacy side effect, and synergy prediction

        \item \textbf{PathMe}\hfill\href{https://github.com/PathwayMerger/PathMe}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} PathwayMerger/PathMe}
\\
    Web application for exploration of pathway databases

        \item \textbf{INDRA}\hfill\href{https://github.com/sorgerlab/indra}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} sorgerlab/indra}
\\
    Automated knowledge assembly and modeling in biomedicine

        \item \textbf{INDRA CoGEx}\hfill\href{https://github.com/bgyori/indra\_cogex}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} bgyori/indra\_cogex}
\\
    A $10^8$ relation-scale knowledge graph extending on causal knowledge from INDRA

        \item \textbf{MIRA}\hfill\href{https://github.com/indralab/mira}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} indralab/mira}
\\
    Machine-assisted scientific modeling using meta-model templates and domain knowledge graphs

        \item \textbf{Gilda}\hfill\href{https://github.com/indralab/gilda}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} indralab/gilda}
\\
    Biomedical named entity recognition and grounding using machine-learned disambiguation

    \end{enumerate}

    \section{Databases}\label{sec:databases}
    \begin{enumerate}
    \itemsep0.5em
        \item \textbf{Bioregistry}\hfill\href{https://github.com/bioregistry/bioregistry}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} bioregistry/bioregistry}
, \url{https://bioregistry.io}\\
    An integrative meta-registry of biological databases, ontologies, and nomenclatures

        \item \textbf{Biomappings}\hfill\href{https://github.com/biopragmatics/biomappings}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} biopragmatics/biomappings}
, \url{https://biopragmatics.github.io/biomappings}\\
    Predicted and curated mappings between named biological entities

        \item \textbf{Biolookup}\hfill\href{https://github.com/biopragmatics/biolookup}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} biopragmatics/biolookup}
, \url{http://biolookup.io}\\
    Comprehensive database of identifiers, names, synonyms, cross-references, properties, and relations for biomedical entities

        \item \textbf{Bioversions}\hfill\href{https://github.com/biopragmatics/bioversions}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} biopragmatics/bioversions}
\\
    Automated tracking of the current version for each biological database?

        \item \textbf{OBO Database Ingest}\hfill\href{https://github.com/biopragmatics/obo-db-ingest}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} biopragmatics/obo-db-ingest}
\\
    Conversion of biomedical databases into ontologies

        \item \textbf{Chemical Roles Graph}\hfill\href{https://github.com/chemical-roles/chemical-roles}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} chemical-roles/chemical-roles}
\\
    Connecting roles in the ChEBI ontology to their targets

        \item \textbf{CONSO}\hfill\href{https://github.com/pharmacome/conso}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} pharmacome/conso}
\\
    Ontology of phenomena related to neurodegeneration

        \item \textbf{CONIB}\hfill\href{https://github.com/pharmacome/conib}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} pharmacome/conib}
\\
    Curated knowledge graphs describing neurodegeneration in BEL

        \item \textbf{ComPath}\hfill\href{https://github.com/ComPath/ComPath}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} ComPath/ComPath}
, \url{https://compath.scai.fraunhofer.de/}\\
    A data integration pipeline, manually curated dataset of mappings between pathway databases, and web application for integrative analysis of pathways

    \end{enumerate}

    \subsection{External Contributions}\label{sec:contributions}
        \wikidata{Q114667913}{EHDAA2},        \wikidata{Q4866972}{BFO},        \wikidata{Q135085}{GO},        \wikidata{Q28445410}{ECO},        \wikidata{Q113012675}{OMO},        \wikidata{Q55118393}{FYPO},        \wikidata{Q113006456}{COB},        \wikidata{Q28729320}{RO},        \wikidata{Q81661653}{HSAPDV},        \wikidata{Q81661698}{MMUSDV},        \wikidata{Q81661707}{MP},        \wikidata{Q81661533}{AGRO},        \wikidata{Q110998476}{CIDO},        \wikidata{Q81661607}{EUPATH},        \wikidata{Q81661647}{HANCESTRO},        \wikidata{Q81661730}{OBA},        \wikidata{Q81661775}{PDUMDV},        \wikidata{Q81661641}{GEO},        \wikidata{Q81661655}{HTN},        \wikidata{Q81661821}{TTO},        \wikidata{Q81661700}{MOD},        \wikidata{Q55118285}{CL},        \wikidata{Q55118328}{CDAO},        \wikidata{Q112972813}{AISM},        \wikidata{Q81661648}{HAO},        \wikidata{Q113014431}{ONS},        \wikidata{Q113014435}{PCL},        \wikidata{Q113014439}{SWO},        \wikidata{Q81661818}{TO},        \wikidata{Q81661813}{TAXRANK},        \wikidata{Q81661834}{WBBT},        \wikidata{Q81661837}{WBLS},        \wikidata{Q81661748}{OLATDV},        \wikidata{Q81661777}{PECO},        \wikidata{Q81661827}{UPHENO},        \wikidata{Q81661839}{XCO},        \wikidata{Q55118415}{IAO},        \wikidata{Q7201529}{PO},        \wikidata{Q55118512}{OGMS},        \wikidata{Q55118561}{OHD},        \wikidata{Q7095051}{OBI},        \wikidata{Q81661690}{MIAPA},        \wikidata{Q27468140}{MONDO},        \wikidata{Q113012667}{LEPAO},        \wikidata{Q113012658}{EXO},        \wikidata{Q81661546}{BSPO},        \wikidata{Q81661585}{DDPHENO},        \wikidata{Q7876491}{UBERON},        \wikidata{Q81661620}{FBBT},        \wikidata{Q31110555}{ENVO},        \wikidata{Q81661810}{SYMP},        \wikidata{Q55118301}{CHMO},        \wikidata{Q81661725}{OAE},        \wikidata{Q17027854}{HP},        \wikidata{Q113012668}{MAXO},        \wikidata{Q113009128}{COLAO},        \wikidata{Q108394480}{SSSOM}


\section{Funding}
    \begin{enumerate}
    \itemsep0.75em
    \item  \textbf{Rapid Assessment of Platform Technologies to Expedite Response}\\
        Defense Threat Reduction Agency (DTRA), (FP00012844), \$623K (subcontract) / \$15M overall, 2022-2024 (18 months), Grant: HDTRA1242031\\
        Role: Key Person. Co-wrote proposal with PNNL collaborators. Responsible for design and implementation of research plan.
    \item  \textbf{MIRA: Modeling with an Intelligent Reasoning Assistant}\\
        DARPA ASKEM Program, (HR00112220036), \$2.1M, 2022-2026 (42 months)\\
        Role: Participant
    \item  \textbf{Young Faculty Award / DARPA Director’s Fellowship Award}\\
        DARPA, (W911NF2010255), \$750K, 2020-2023 (36 months)\\
        Role: Participant
    \item (past) \textbf{Ecosystem of Machine-maintained Models with Automated Assembly}\\
        DARPA Automating Scientific Knowledge Extraction (ASKE) / Artificial Intelligence Exploration (AIE) program, (HR00111990009), \$2M, 2018-2020 (18 months + extension)\\
        Role: Participant
    \item (past) \textbf{PYBEL2NDEX}\\
        University of California, San Diego, \$28K 2018 (6 months)\\
        Role: Key person. Wrote proposal. Responsible for design and implementation of research plan.
    \end{enumerate}

\hfill \break
{\huge Community}

    \section{Professional Affiliations}
    \begin{itemize}
        \item     \href{https://bioregistry.io/wikidata:Q23809291}{International Society of Biocuration} (2021 -)

        \item     \href{https://bioregistry.io/wikidata:Q95587147}{CoronaWhy} (2020 - 2021)

        \item     \href{https://bioregistry.io/wikidata:}{OpenBEL Consortium} (2017 -)

        \item     \href{https://bioregistry.io/wikidata:Q655757}{Erasmus Student Network Bonn} (2016 -)

        \item     \href{https://bioregistry.io/wikidata:Q247556}{American Chemical Society} (2011 - 2022)

    \end{itemize}

\section{Service to the Community}

\subsection{Scholarly Journals}

\begin{enumerate}
\item Scientific article reviewer in:
\begin{itemize}
    \item Bioinformatics
    \item Database
    \item BMC Bioinformatics
    \item Journal of Cheminformatics
    \item Journal of Biomedical Semantics
    \item eLife
    \item MATCH Communications in Mathematical and in Computer Chemistry
\end{itemize}
\item Reproducibility Editor, Journal of Cheminformatics (pending)
\end{enumerate}

    \subsection{Conference Organizing Committees}
    \begin{enumerate}
        \item \href{https://bioregistry.io/wikidata:Q111430238?provider=scholia}{Biocuration 2023} (Co-chair)
        \item \href{https://biopragmatics.github.io/workshops/WPCI2022.html}{2022 Workshop on Prefixes, CURIEs, and IRIs} (Organizer)
        \item \href{https://icbo-conference.github.io/icbo2022/workshops-and-tutorials/#workshop-on-ontology-tools-and-workflows}{ICBO 2022 Workshop on Ontology Tools} (Co-organizer)
        \item \href{https://icbo-conference.github.io/icbo2022/program-committee}{ICBO 2022} (Program Committee)
        \item \href{https://www.iscb.org/ismb2022-program/abstracts/bio-ontologies}{ISMB 2022 (Bio-Ontologies/BOSC joint session)} (Program Committee)
        \item \href{https://bioregistry.io/wikidata:Q109407979?provider=scholia}{Biocuration 2022} (Organizing Committee)
        \item \href{https://biopragmatics.github.io/workshops/WPCI2021.html}{2021 Workshop on Prefixes, CURIEs, and IRIs} (Organizer)
    \end{enumerate}

\hfill \break
{\huge Teaching}

    \section{Courses Taught}
        \subsection{University of Bonn}
        \begin{enumerate}
        \item     Mechanism Enrichment Using NeuroMMSig (Practical; Winter 2020-2021)

        \item     Mechanism Enrichment Using NeuroMMSig (Practical; Winter 2019-2020)

        \item     Mathematics Meets Life Sciences (Lecture; Winter 2019-2020)

        \item     Enzyme Technology Internship (Practical; Summer 2019)

        \item     Life Sciences Knowledge Discovery (Lecture; Summer 2019)

        \item     Knowledge Assembly, Data Integration, and Modeling in Systems and Networks Biology (Seminar; Winter 2018-2019)

        \item     Biological Databases (Lecture; Winter 2018-2019)

        \item     Life Sciences Knowledge Discovery (Lecture; Summer 2018)

        \item     Biological Databases (Lecture; Winter 2017-2018)

        \item     Life Sciences Knowledge Discovery (Lecture; Summer 2017)

        \item     Biomedical Database Lab (Practical; Winter 2016-2017)

        \end{enumerate}
        \subsection{Northeastern University}
        \begin{enumerate}
        \item     Drug Discovery and Development (Lecture; Summer II 2015)

        \item     Organic Chemistry II for Majors (Lecture; Spring 2015)

        \item     Organic Chemistry I for Majors (Lecture; Fall 2014)

        \item     Organic Chemistry II for Majors (Lecture; Spring 2014)

        \item     Organic Chemistry I for Majors (Lecture; Fall 2013)

        \end{enumerate}

    \section{Supervision}
    \subsection{CoronaWhy}
        \begin{itemize}
        \item     Aman Choudhri {\scriptsize     \href{https://orcid.org/0000-0003-4963-6651}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (June - October 2020)

        \end{itemize}
    \subsection{Fraunhofer}
        \begin{itemize}
        \item     Lauren Nicole DeLong {\scriptsize     \href{https://orcid.org/0000-0001-6618-3370}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (September - December 2019)

        \item     Vinay Bharadhwaj {\scriptsize     \href{https://orcid.org/0000-0002-9901-9494}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (July - December 2019)

        \item     Yojana Gadiya {\scriptsize     \href{https://orcid.org/0000-0002-7683-0452}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (April - May 2019)

        \item     Trusha Adeshara {\scriptsize     \href{https://orcid.org/0000-0002-8929-4724}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (April - May 2019)

        \item     Rana Aldisi {\scriptsize     \href{https://orcid.org/0000-0002-3034-9970}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (July 2018 - March 2019)

        \item     Lingling Xu {\scriptsize     \href{https://orcid.org/0000-0002-0303-8616}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (July 2018 - March 2019)

        \item     Kristian Kolpeja {\scriptsize     \href{https://orcid.org/0000-0001-9661-5277}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (July - November 2018)

        \item     Esther Wollert {\scriptsize     \href{https://orcid.org/0000-0002-7128-929X}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (July 2018 - August 2019)

        \item     Sandra Spalek {\scriptsize     \href{https://orcid.org/0000-0002-6117-4413}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (July 2018 - August 2019)

        \item     Keerthika Lohanadan     Student Research Assistant (July - September 2018)

        \item     Colin Birkenbihl {\scriptsize     \href{https://orcid.org/0000-0002-7212-7700}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Student Research Assistant (July - October 2017)

        \item     Aram Grigoryan     Student Research Assistant (July - December 2017)

        \end{itemize}
    \subsection{University of Bonn}
        \begin{itemize}
        \item     Mauricio Pio de Lacerda {\scriptsize     \href{https://orcid.org/0000-0002-1770-3007}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Master's Student (March - December 2019)

        \item     Rana Aldisi {\scriptsize     \href{https://orcid.org/0000-0002-3034-9970}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Master's Student (March - December 2019)

        \item     Lingling Xu {\scriptsize     \href{https://orcid.org/0000-0002-0303-8616}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Master's Student (March - December 2019)

        \item     Özlem Muslu {\scriptsize     \href{https://orcid.org/0000-0003-0408-6190}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16}}
}    Master's Student (May - December 2018)

        \end{itemize}

\end{document}