
\documentclass[11pt,a4paper,sans]{moderncv} % Font sizes: 10, 11, or 12; paper sizes: a4paper, letterpaper, a5paper, legalpaper, executivepaper or landscape; font families: sans or roman

\moderncvstyle{classic} % CV theme - options include: 'casual' (default), 'classic', 'oldstyle' and 'banking'
\moderncvcolor{blue} % CV color - options include: 'blue' (default), 'orange', 'green', 'red', 'purple', 'grey' and 'black'

\usepackage{hyperref}
\usepackage{graphicx}

\usepackage[scale=0.8]{geometry} % Reduce document margins
\setlength{\hintscolumnwidth}{1.5cm} % Uncomment to change the width of the dates column
\setlength{\makecvtitlenamewidth}{10cm} % For the 'classic' style, uncomment to adjust the width of the space allocated to your name


\newcommand{\curie}[2]{\href{https://bioregistry.io/#1:#2}{{#1:#2}}}
\newcommand{\wikidata}[2]{\href{https://bioregistry.io/wikidata:#1}{{#2}}}

\begin{document}

{\Huge Charles Tapley Hoyt}

\vspace{3mm}

\href{https://cthoyt.com}{cthoyt.com}
$\cdot$
\href{https://linkedin.com/in/cthoyt}{\includegraphics[scale=0.055]{img/LI-In-Bug} linkedin.com/in/cthoyt}
$\cdot$
\href{https://github.com/cthoyt}{\includegraphics[scale=0.25]{img/GitHub-Mark-32px} github.com/cthoyt}
$\cdot$
\href{https://twitter.com/cthoyt}{\includegraphics[scale=0.02]{img/Twitter_Social_Icon_Circle_Color} @ cthoyt}
$\cdot$
\href{https://bioregistry.io/wikidata:Q47475003}{\includegraphics[scale=0.01]{img/wikidata_logo} Q47475003}
\\
\href{https://orcid.org/0000-0003-4423-4370}{\includegraphics[scale=0.5]{img/ORCIDiD_icon16x16} https://orcid.org/0000-0003-4423-4370}
$\cdot$
\href{https://scholar.google.com/citations?user=PjrpzUIAAAAJ}{\includegraphics[scale=0.015625]{img/google_scholar_icon_130918} PjrpzUIAAAAJ}

\section{Fields of Work}\label{sec:fields}

    \href{https://bioregistry.io/wikidata:Q128570}{Bioinformatics},     \href{https://bioregistry.io/wikidata:Q25303877}{Pathway Analysis},     \href{https://bioregistry.io/wikidata:Q2539}{Machine Learning},     \href{https://bioregistry.io/wikidata:Q30642}{Natural Language Processing},     \href{https://bioregistry.io/wikidata:Q324254}{Ontology},     \href{https://bioregistry.io/wikidata:Q33002955}{Knowledge Graph},     \href{https://bioregistry.io/wikidata:Q50327360}{Proteochemometrics},     \href{https://bioregistry.io/wikidata:Q5308921}{Drug Repositioning},     \href{https://bioregistry.io/wikidata:Q815297}{Systems Biology},     \href{https://bioregistry.io/wikidata:Q910164}{Cheminformatics}
    \section{Software}\label{sec:software}
        \cventry{}{SSSOM-py}{Python package}{}{}{\url{https://github.com/mapping-commons/sssom-py/}, \url{https://mapping-commons.github.io/sssom-py/}}
        \cventry{}{PyBEL}{Python library for Biological Expression Language}{}{}{\url{https://github.com/pybel/pybel}, \url{https://pybel.github.io}}
        \cventry{}{Integrated Network and Dynamical Reasoning Assembler}{Text mining and systems biology modeling software}{}{}{\url{https://github.com/sorgerlab/indra}, \url{https://www.indra.bio/}}
        \cventry{}{PyKEEN}{Learning, evaluation and applications of knowledge graph embeddings.}{}{}{\url{https://github.com/pykeen/pykeen}, \url{https://pykeen.github.io}}

    \section{Spoken Languages}\label{sec:languages}

        \href{https://bioregistry.io/wikidata:Q1860}{English},
        \href{https://bioregistry.io/wikidata:Q188}{German}
\section{Articles}\label{articles}
    \subsection{2022}
    \begin{enumerate}
    \itemsep0.5em
        \item
            \wikidata{Q113009731}{\textbf{Unifying the Identification of Biomedical Entities with the Bioregistry}}.
\\ \href{https://bioregistry.io/doi:10.1101/2022.07.08.499378}{DOI:10.1101/2022.07.08.499378}} \href{https://bioregistry.io/biorxiv:2022.07.08.499378}{bioRxiv:2022.07.08.499378}}
        \item
            \wikidata{Q112942500}{\textbf{Ontology Development Kit: a toolkit for building, maintaining, and standardising biomedical ontologies}}.
\\ \href{https://bioregistry.io/arxiv:2207.02056}{arXiv:2207.02056}}
        \item
            \wikidata{Q111697653}{\textbf{ProtSTonKGs: A Sophisticated Transformer Trained on Protein Sequences, Text, and Knowledge Graphs}}.

        \item
            \wikidata{Q111337218}{\textbf{A Unified Framework for Rank-based Evaluation Metrics for Link Prediction in Knowledge Graphs}}.
\\ \href{https://bioregistry.io/arxiv:2203.07544}{arXiv:2203.07544}}
        \item
            \wikidata{Q113227575}{\textbf{PyBioPAX: biological pathway exchange in Python}}.
\\ \href{https://bioregistry.io/doi:10.21105/JOSS.04136}{DOI:10.21105/joss.04136}}
        \item
            \wikidata{Q111102996}{\textbf{An Open Challenge for Inductive Link Prediction on Knowledge Graphs}}.
\\ \href{https://bioregistry.io/arxiv:2203.01520}{arXiv:2203.01520}}
        \item
            \wikidata{Q110889136}{\textbf{ChemicalX: A Deep Learning Library for Drug Pair Scoring}}.
\\ \href{https://bioregistry.io/arxiv:2202.05240}{arXiv:2202.05240}}
        \item
            \wikidata{Q112063219}{\textbf{STonKGs: A Sophisticated Transformer Trained on Biomedical Text and Knowledge Graphs}}.
\\ \href{https://bioregistry.io/pubmed:34986221}{PMID:34986221} \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTAC001}{DOI:10.1093/bioinformatics/btac001}} \href{https://bioregistry.io/biorxiv:2021.08.17.456616}{bioRxiv:2021.08.17.456616}}
    \end{enumerate}
    \subsection{2021}
    \begin{enumerate}
    \itemsep0.5em
        \item
            \wikidata{Q110123027}{\textbf{A Simple Standard for Sharing Ontological Mappings (SSSOM)}}.
\\ \href{https://bioregistry.io/arxiv:2112.07051}{arXiv:2112.07051}}
        \item
            \wikidata{Q108525107}{\textbf{Gilda: biomedical entity text normalization with machine-learned disambiguation as a service}}.
\\ \href{https://bioregistry.io/doi:10.1101/2021.09.10.459803}{DOI:10.1101/2021.09.10.459803}} \href{https://bioregistry.io/biorxiv:2021.09.10.459803}{bioRxiv:2021.09.10.459803}}
        \item
            \wikidata{Q110894741}{\textbf{The role of metadata in reproducible computational research}}.
\\ \href{https://bioregistry.io/pubmed:34553169}{PMID:34553169} \href{https://bioregistry.io/doi:10.1016/J.PATTER.2021.100322}{DOI:10.1016/j.patter.2021.100322}} \href{https://bioregistry.io/arxiv:2006.08589}{arXiv:2006.08589}}
        \item
            \wikidata{Q108016409}{\textbf{Integrating multi-omics data reveals function and therapeutic potential of deubiquitinating enzymes}}.
\\ \href{https://bioregistry.io/doi:10.1101/2021.08.06.455458}{DOI:10.1101/2021.08.06.455458}} \href{https://bioregistry.io/biorxiv:2021.08.06.455458}{bioRxiv:2021.08.06.455458}}
        \item
            \wikidata{Q107296731}{\textbf{Wavelet-Packet Powered Deepfake Image Detection}}.
\\ \href{https://bioregistry.io/arxiv:2106.09369}{arXiv:2106.09369}}
        \item
            \wikidata{Q107296776}{\textbf{Understanding the Performance of Knowledge Graph Embeddings in Drug Discovery}}.
\\ \href{https://bioregistry.io/arxiv:2105.10488}{arXiv:2105.10488}}
        \item
            \wikidata{Q106327616}{\textbf{A Review of Biomedical Datasets Relating to Drug Discovery: A Knowledge Graph Perspective}}.
\\ \href{https://bioregistry.io/arxiv:2102.10062}{arXiv:2102.10062}}
        \item
            \wikidata{Q108126839}{\textbf{A Systems Biology Approach for Hypothesizing the Effect of Genetic Variants on Neuroimaging Features in Alzheimerâ€™s Disease}}.
\\ \href{https://bioregistry.io/pubmed:33554913}{PMID:33554913} \href{https://bioregistry.io/doi:10.3233/JAD-201397}{DOI:10.3233/jad-201397}}
        \item
            \wikidata{Q109583780}{\textbf{PyKEEN 1.0: A Python Library for Training and Evaluating Knowledge Graph Embeddings}}.

        \item
            \wikidata{Q104846171}{\textbf{Leveraging Structured Biological Knowledge for Counterfactual Inference: a Case Study of Viral Pathogenesis}}.
\\ \href{https://bioregistry.io/arxiv:2101.05136}{arXiv:2101.05136}}
    \end{enumerate}
    \subsection{2020}
    \begin{enumerate}
    \itemsep0.5em
        \item
            \wikidata{Q104920313}{\textbf{CLEP: A Hybrid Data- and Knowledge- Driven Framework for Generating Patient Representations}}.
\\ \href{https://bioregistry.io/doi:10.1101/2020.08.20.259226}{DOI:10.1101/2020.08.20.259226}} \href{https://bioregistry.io/biorxiv:2020.08.20.259226}{bioRxiv:2020.08.20.259226}}
        \item
            \wikidata{Q96909013}{\textbf{Extension of Roles in the ChEBI Ontology}}.
\\ \href{https://bioregistry.io/doi:10.26434/CHEMRXIV.12591221}{DOI:10.26434/chemrxiv.12591221}}
        \item
            \wikidata{Q97063807}{\textbf{The Minimum Information about a Molecular Interaction Causal Statement (MI2CAST)}}.
\\ \href{https://bioregistry.io/pubmed:32637990}{PMID:32637990} \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTAA622}{DOI:10.1093/bioinformatics/btaa622}}
        \item
            \wikidata{Q96631308}{\textbf{Bringing Light Into the Dark: A Large-scale Evaluation of Knowledge Graph Embedding Models Under a Unified Framework}}.
\\ \href{https://bioregistry.io/arxiv:2006.13365}{arXiv:2006.13365}}
        \item
            \wikidata{Q98200725}{\textbf{GuiltyTargets: Prioritization of Novel Therapeutic Targets with Deep Network Representation Learning}}.
\\ \href{https://bioregistry.io/pubmed:32750869}{PMID:32750869} \href{https://bioregistry.io/doi:10.1109/TCBB.2020.3003830}{DOI:10.1109/tcbb.2020.3003830}} \href{https://bioregistry.io/biorxiv:521161}{bioRxiv:521161}}
        \item
            \wikidata{Q96169295}{\textbf{PS4DR: a multimodal workflow for identification and prioritization of drugs based on pathway signatures}}.
\\ \href{https://bioregistry.io/pubmed:32503412}{PMID:32503412} \href{https://bioregistry.io/doi:10.1186/S12859-020-03568-5}{DOI:10.1186/s12859-020-03568-5}}
        \item
            \wikidata{Q90643177}{\textbf{A Computational Approach for Mapping Heme Biology in the Context of Hemolytic Disorders}}.
\\ \href{https://bioregistry.io/pubmed:32211383}{PMID:32211383} \href{https://bioregistry.io/doi:10.3389/FBIOE.2020.00074}{DOI:10.3389/fbioe.2020.00074}} \href{https://bioregistry.io/biorxiv:804906}{bioRxiv:804906}}
    \end{enumerate}
    \subsection{2019}
    \begin{enumerate}
    \itemsep0.5em
        \item
            \wikidata{Q91869190}{\textbf{The Impact of Pathway Database Choice on Statistical Enrichment Analysis and Predictive Modeling}}.
\\ \href{https://bioregistry.io/pubmed:31824580}{PMID:31824580} \href{https://bioregistry.io/doi:10.3389/FGENE.2019.01203}{DOI:10.3389/fgene.2019.01203}} \href{https://bioregistry.io/biorxiv:654442}{bioRxiv:654442}}
        \item
            \wikidata{Q90659732}{\textbf{Quantifying mechanisms in neurodegenerative diseases (NDDs) using candidate mechanism perturbation amplitude (CMPA) algorithm}}.
\\ \href{https://bioregistry.io/pubmed:31604427}{PMID:31604427} \href{https://bioregistry.io/doi:10.1186/S12859-019-3101-1}{DOI:10.1186/s12859-019-3101-1}}
        \item
            \wikidata{Q72584451}{\textbf{The KEEN Universe}}.
\\ \href{https://bioregistry.io/doi:10.1007/978-3-030-30796-7_1}{DOI:10.1007/978-3-030-30796-7_1}}
        \item
            \wikidata{Q72621157}{\textbf{Predicting Missing Links Using PyKEEN}}.

        \item
            \wikidata{Q69105793}{\textbf{RatVec: A General Approach for Low-dimensional Distributed Vector Representations via Domain-specific Rational Kernels}}.

        \item
            \wikidata{Q63709729}{\textbf{BioKEEN: a library for learning and evaluating biological knowledge graph embeddings}}.
\\ \href{https://bioregistry.io/pubmed:30768158}{PMID:30768158} \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTZ117}{DOI:10.1093/bioinformatics/btz117}} \href{https://bioregistry.io/biorxiv:475202}{bioRxiv:475202}}
        \item
            \wikidata{Q64054035}{\textbf{PathMe: merging and exploring mechanistic pathway knowledge}}.
\\ \href{https://bioregistry.io/pubmed:31092193}{PMID:31092193} \href{https://bioregistry.io/doi:10.1186/S12859-019-2863-9}{DOI:10.1186/s12859-019-2863-9}} \href{https://bioregistry.io/biorxiv:451625}{bioRxiv:451625}}
        \item
            \wikidata{Q63709723}{\textbf{Integration of Structured Biological Data Sources using Biological Expression Language}}.
\\ \href{https://bioregistry.io/doi:10.1101/631812}{DOI:10.1101/631812}} \href{https://bioregistry.io/biorxiv:631812}{bioRxiv:631812}}
        \item
            \wikidata{Q60045097}{\textbf{ComPath: an ecosystem for exploring, analyzing, and curating mappings across pathway databases}}.
\\ \href{https://bioregistry.io/pubmed:30564458}{PMID:30564458} \href{https://bioregistry.io/doi:10.1038/S41540-018-0078-8}{DOI:10.1038/s41540-018-0078-8}} \href{https://bioregistry.io/biorxiv:353235}{bioRxiv:353235}}
        \item
            \wikidata{Q64911025}{\textbf{Re-curation and rational enrichment of knowledge graphs in Biological Expression Language}}.
\\ \href{https://bioregistry.io/pubmed:31225582}{PMID:31225582} \href{https://bioregistry.io/doi:10.1093/DATABASE/BAZ068}{DOI:10.1093/database/baz068}} \href{https://bioregistry.io/biorxiv:536409}{bioRxiv:536409}}
        \item
            \wikidata{Q82511885}{\textbf{Challenges of Integrative Disease Modeling in Alzheimer&#39;s Disease}}.
\\ \href{https://bioregistry.io/pubmed:31993440}{PMID:31993440} \href{https://bioregistry.io/doi:10.3389/FMOLB.2019.00158}{DOI:10.3389/fmolb.2019.00158}}
    \end{enumerate}
    \subsection{2018}
    \begin{enumerate}
    \itemsep0.5em
        \item
            \wikidata{Q60302045}{\textbf{BEL Commons: an environment for exploration and analysis of networks encoded in Biological Expression Language}}.
\\ \href{https://bioregistry.io/pubmed:30576488}{PMID:30576488} \href{https://bioregistry.io/doi:10.1093/DATABASE/BAY126}{DOI:10.1093/database/bay126}} \href{https://bioregistry.io/biorxiv:288274}{bioRxiv:288274}}
        \item
            \wikidata{Q51056857}{\textbf{BEL2ABM: agent-based simulation of static models in Biological Expression Language}}.
\\ \href{https://bioregistry.io/pubmed:29949955}{PMID:29949955} \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTY107}{DOI:10.1093/bioinformatics/bty107}}
        \item
            \wikidata{Q42695788}{\textbf{PyBEL: a Computational Framework for Biological Expression Language}}.
\\ \href{https://bioregistry.io/pubmed:29048466}{PMID:29048466} \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTX660}{DOI:10.1093/bioinformatics/btx660}}
        \item
            \wikidata{Q55315340}{\textbf{A systematic approach for identifying shared mechanisms in epilepsy and its comorbidities}}.
\\ \href{https://bioregistry.io/pubmed:29873705}{PMID:29873705} \href{https://bioregistry.io/doi:10.1093/DATABASE/BAY050}{DOI:10.1093/database/bay050}} \href{https://bioregistry.io/biorxiv:269860}{bioRxiv:269860}}
    \end{enumerate}
    \subsection{2017}
    \begin{enumerate}
    \itemsep0.5em
        \item
            \wikidata{Q47479157}{\textbf{A Case Study on the Parametric Occurrence of Multiple Steady States}}.
\\ \href{https://bioregistry.io/doi:10.1145/3087604.3087622}{DOI:10.1145/3087604.3087622}} \href{https://bioregistry.io/arxiv:1704.08997}{arXiv:1704.08997}}
    \end{enumerate}
    \subsection{2014}
    \begin{enumerate}
    \itemsep0.5em
        \item
            \wikidata{Q34138086}{\textbf{Repurposing human PDE4 inhibitors for neglected tropical diseases: design, synthesis and evaluation of cilomilast analogues as Trypanosoma brucei PDEB1 inhibitors}}.
\\ \href{https://bioregistry.io/pubmed:25127163}{PMID:25127163} \href{https://bioregistry.io/doi:10.1016/J.BMCL.2014.07.063}{DOI:10.1016/j.bmcl.2014.07.063}}
    \end{enumerate}

        \section{Reviews}\label{sec:reviews}
        \begin{enumerate}
        \itemsep0.5em
            \item
                \wikidata{Q112143478}{\textbf{The LOTUS initiative for open knowledge management in natural products research}} (2022).
\\ \href{https://bioregistry.io/pubmed:35616633}{PMID:35616633} \href{https://bioregistry.io/doi:10.7554/ELIFE.70780}{DOI:10.7554/elife.70780}}
            \item
                \wikidata{Q112248015}{\textbf{New reasons for biologists to write with a formal language}} (2022).
\\ \href{https://bioregistry.io/doi:10.1093/DATABASE/BAAC039}{DOI:10.1093/database/baac039}}
            \item
                \wikidata{Q107361978}{\textbf{PecanPy: a fast, efficient and parallelized Python implementation of node2vec}} (2021).
\\ \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTAB202}{DOI:10.1093/bioinformatics/btab202}}
        \end{enumerate}


        \section{Acknowledgements}\label{sec:acknowledgements}
        \begin{enumerate}
        \itemsep0.5em
            \item
                \wikidata{Q113662614}{\textbf{Automated assembly of molecular mechanisms at scale from text mining and curated databases}} (2022).
\\ \href{https://bioregistry.io/doi:10.1101/2022.08.30.505688}{DOI:10.1101/2022.08.30.505688}} \href{https://bioregistry.io/biorxiv:2022.08.30.505688}{bioRxiv:2022.08.30.505688}}
            \item
                \wikidata{Q107361978}{\textbf{PecanPy: a fast, efficient and parallelized Python implementation of node2vec}} (2021).
\\ \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTAB202}{DOI:10.1093/bioinformatics/btab202}}
            \item
                \wikidata{Q104619209}{\textbf{The status of causality in biological databases: data resources and data retrieval possibilities to support logical modeling}} (2020).
\\ \href{https://bioregistry.io/pubmed:33378765}{PMID:33378765} \href{https://bioregistry.io/doi:10.1093/BIB/BBAA390}{DOI:10.1093/bib/bbaa390}}
            \item
                \wikidata{Q38708528}{\textbf{Multimodal Mechanistic Signatures for Neurodegenerative Diseases (NeuroMMSig): a web server for mechanism enrichment}} (2017).
\\ \href{https://bioregistry.io/pubmed:28651363}{PMID:28651363} \href{https://bioregistry.io/doi:10.1093/BIOINFORMATICS/BTX399}{DOI:10.1093/bioinformatics/btx399}}
        \end{enumerate}


\end{document}